### PR TITLE
Add authors

### PR DESCRIPTION
**Pull request by santisoler**
Feel free to add yourself as an author. You can leave your name and affiliation in a comment, I can take care of adding you to the authors list.

- Add Lu Li as author
- Add Santi Soler as author

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
 
 Fatiando a Terra Project,
 Leonardo Uieda<sup>1</sup>,
-ADD YOUR NAME AND AFFILIATION HERE
+Lu Li<sup>2</sup>,
+Santiago Soler<sup>3,4</sup>
 
 > <sup>1</sup>Department of Earth, Ocean and Ecological Sciences, School of Environmental Sciences, University of Liverpool, UK
 > <br>
+> <sup>2</sup> School of Earth Sciences, The University of Western Australia, Australia
```
Comment (leouieda) [inline README.md:9]:
```suggestion
> <sup>2</sup> School of Earth Sciences, The University of Western Australia, Australia
> <br>
```

File: README.md [hunk 2]
```diff
@@ -2,10 +2,14 @@
...
+> <sup>3</sup> CONICET, Argentina
```
Comment (leouieda) [inline README.md:11]:
```suggestion
> <sup>3</sup> CONICET, Argentina
> <br>
```

Comment (leouieda) [reply README.md:11]: A very patriotic review 🙂

**Comment by leouieda** [comment]
@LL-Geo if you have an ORCID I can add that to the system as well.

**Comment by leouieda** [comment]
Is this you? https://orcid.org/0000-0001-6165-2828

**Comment by leouieda** [comment]
... aaand AGU also requires an email address to add people as authors. Feel free to post that here or send privately on Slack if preferred.

**Comment by LL-Geo** [comment]
> Is this you? https://orcid.org/0000-0001-6165-2828

Yup~ 😄

**Comment by LL-Geo** [comment]
> ... aaand AGU also requires an email address to add people as authors. Feel free to post that here or send privately on Slack if preferred.

My email address is lu.li@research.uwa.edu.au Thanks 👍

File: README.md (hunk 1 above)
Comment (santisoler) [reply README.md:9]: Forgot the line breaks!

**Pull request merged**